EECS3311-PD1/src/main/resources/fxml/ReportView.fxml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.TextArea?>
+
+<?import javafx.scene.layout.VBox?>
+<fx:root type="VBox" xmlns="http://javafx.com/javafx/19" xmlns:fx="http://javafx.com/fxml/1">
+      <TextArea fx:id="textAreaReport" editable="false" prefHeight="400.0" prefWidth="600.0" />
```

Comment:
Try to see if you really need attributes like `prefHeight` or `prefWidth` and remove them if they're redundant